backend/package.json

```diff
@@ -25,27 +25,35 @@
     "@nestjs/common": "^9.0.0",
     "@nestjs/config": "^2.2.0",
     "@nestjs/core": "^9.0.0",
+    "@nestjs/jwt": "^10.0.1",
     "@nestjs/mapped-types": "1.2.0",
+    "@nestjs/passport": "^9.0.0",
     "@nestjs/platform-express": "^9.0.0",
     "@nestjs/platform-socket.io": "^9.2.1",
     "@nestjs/typeorm": "^9.0.1",
     "@nestjs/websockets": "^9.2.1",
-    "class-validator": "^0.13",
+    "bcrypt": "^5.1.0",
+    "class-transformer": "^0.5.1",
+    "class-validator": "^0.13.2",
     "dotenv": "^16.0.3",
     "dotenv-expand": "^10.0.0",
+    "passport": "^0.6.0",
+    "passport-jwt": "^4.0.1",
+    "passport-local": "^1.0.0",
     "pg": "^8.8.0",
     "reflect-metadata": "^0.1.13",
-    "rimraf": "^3.0.2",
-    "rxjs": "^7.8.0",
     "socket.io": "^4.5.4",
     "typeorm": "^0.3.11"
   },
   "devDependencies": {
     "@nestjs/schematics": "^9.0.4",
     "@nestjs/testing": "^9.0.0",
+    "@types/bcrypt": "^5.0.0",
     "@types/express": "^4.17.15",
     "@types/jest": "^29.2.4",
     "@types/node": "^18.11.18",
+    "@types/passport-jwt": "^3.0.8",
+    "@types/passport-local": "^1.0.35",
```

Comment:
I don't like imports with "or newer" (^) version. There could be compatibilty problems in the future.

Reply:
This import's automaticly, that's good point, i will fix it, I don't watch for that before, but in this project i start to use npm ci and know how to package.lock work :)